log-engine-tuulbox/build.gradle.kts

```diff
@@ -14,10 +14,13 @@ kotlin {
         publishAllLibraryVariants()
     }
     js().browser()
+
     iosX64()
     iosArm32()
     iosArm64()
     macosX64()
+    macosArm64()
```

Comment:
I'd like this to be tested before merging it.
Do you need me to test it, or can you try this out on the sensortag sample app?

Reply:
My sensortag died about a year ago. If you can test it faster than me getting a new one, I'd say go for it.